pkg/autodiscover/autodiscover.go

```diff
@@ -261,6 +266,22 @@ func DoAutoDiscover(config *configuration.TestConfiguration) DiscoveredTestData
 	if err != nil {
 		log.Fatal("Cannot get list of all service accounts, err: %v", err)
 	}
+
+	data.SriovNetworks, err = getSriovNetworks(oc, data.Namespaces)
+	if err != nil {
+		log.Fatal("Cannot get list of sriov networks, err: %v", err)
```

Comment:
If the workloads to test are not using SRIOV or SRIOV operator is not installed in the cluster, the certsuite execution fails just in this point, like in the first DCI job that has failed: https://www.distributed-ci.io/jobs/2651e6f7-cf42-44d2-9c62-2ec7adc6615a

```
[FATAL] [Oct 22 21:37:27.025] [autodiscover.go: 272] Cannot get list of sriov networks, err: the server could not find the requested resource (get sriovnetworks.sriovnetwork.openshift.io)
```

I would say that, in case no SRIOV resources are present in the scenario to test, I'll not make the certsuite to fail.

Reply:
Yeah I'll have to either add some safeguards here or just move this `get` inside the test instead of during autodiscovery.

Reply:
I think this is fixed.